### PR TITLE
DOC: formulate pure SymPy amplitude model

### PR DESCRIPTION
**Pull request by shenvitor**
Formulate the model from the PWA101 tutorial (https://github.com/ComPWA/compwa.github.io/pull/217, see **[TR-033](https://compwa.github.io/report/033)**) with SymPy. This issue only formulates the model and does some simple visualizations. After #36, the model can be visualized over phase space.

> [!TIP]
> See temporary preview [here](https://gluex-nstar--13.org.readthedocs.build/symbolics.html).

### CODE REVIEW

**Comment by shenvitor** [comment]
The idea now is to split this branch to one more branch because one for $\eta \pi p$ and the other is $\Lambda K^+ \pi^0$

**Comment by redeboer** [comment]
So the PR has been split up into
- https://github.com/ComPWA/gluex-nstar/pull/13
- https://github.com/ComPWA/gluex-nstar/pull/28
- https://github.com/ComPWA/gluex-nstar/pull/29

Which ones is ready for review?

**Review by redeboer — CHANGES_REQUESTED** [review]
For the three PRs, could you use an imperative sentence for the PR title?
https://compwa.github.io/develop#commit-conventions
Doesn't have to be conventional (with a category), but just the standard following the standard commit guidelines
https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project

**Comment by shenvitor** [comment]
> For the three PRs, could you use an imperative sentence for the PR title?
> https://compwa.github.io/develop#commit-conventions
> Doesn't have to be conventional (with a category), but just the standard following the standard commit guidelines
> https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project
> 
> Optionally you can enforce that with this workflow
> https://github.com/ComPWA/tensorwaves/blob/94d8a62c60dc8de50b2dd4023d563c735d76add8/.github/workflows/pr-linting.yml
> but that also enforces the use of conventional commit categories.

Okay 🙌

**Comment by shenvitor** [comment]
Dear @redeboer 
This PR is now ready for re-review and merge

File: .github/workflows/ci-docs.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 name: Documentation
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: |-
+    ${{ github.ref != format('refs/heads/{0}', github.event.repository.default_branch) }}
```
Comment (redeboer) [inline .github/workflows/ci-docs.yml:5]: @shenvitor I smuggled in this DX improvement: it aborts CI if additional commits are pushed, in order to save CI time 🌍

**Review by redeboer — APPROVED** [review]
Großartig 🙌

**Pull request merged**